app/models/event_orga.py

```diff
@@ -0,0 +1,32 @@
+from app.models import db
+from app.models.base import SoftDeletionModel
+
+
+class EventOrgaModel(SoftDeletionModel):
+    """Event Orga object table"""
+
+    __tablename__ = 'events_orga'
+
+    id = db.Column(db.Integer, primary_key=True, autoincrement=True)
+    name = db.Column(db.String, nullable=False)
+    starts_at = db.Column(db.DateTime(timezone=True))
+    payment_currency = db.Column(db.String, nullable=False)
```

Comment:
This is data duplication. The use of this endpoint is just to collect data from various other sources. You don't need to add columns in the db for this.